aasrp/static/aasrp/javascript/view-requests.js

```diff
@@ -52,6 +52,20 @@ $(document).ready(() => {
             },
             {
                 data: 'request_code',
+                /**
+                 * Render callback
+                 *
+                 * @param data
+                 * @param type
+                 * @returns {string|*}
+                 */
+                render: (data, type) => {
+                    if (type === 'display') {
+                        return `<span>${data}</span><i class="aa-srp-fa-icon copy-text-fa-icon fa-regular fa-copy ms-2" data-clipboard-text="${data}" data-bs-tooltip="aa-srp" aria-label="Copy code to clipboard" data-bs-original-title="Copy code to clipboard"></i>`;
```

Comment:
Why wrap `${data}` in a `span` but not include the copy icon in this wrapper?

`data-bs-original-title` is unnecessary, `title` is enough, Bootstrap does the rest.

"Copy request code […]" instead of just "Copy code […]"

Please ensure the string is translatable.